content/faq.html

```diff
@@ -224,7 +224,7 @@ <h3 id="do-i-have-to-install-apache-hadoop-to-use-flink">Do I have to install Ap
 Hadoop client libraries by default.</p>
 
 <p>Additionally, we provide a special YARN Enabled download of Flink for
-users with an existing Hadoop YARN cluster. <a href="http://hadoop.apache.org/docs/r2.2.0/hadoop-yarn/hadoop-yarn-site/YARN.html">Apache Hadoop
+users with an existing Hadoop YARN cluster. <a href="https://hadoop.apache.org/docs/r2.7.2/hadoop-yarn/hadoop-yarn-site/YARN.html">Apache Hadoop
```

Comment:
I would suggest to link to the latest stable docs instead, i.e. `http://hadoop.apache.org/docs/stable/hadoop-yarn/hadoop-yarn-site/index.html`

Reply:
Thanks @dawidwys for the suggestion. I updated the PR accordingly.